e0works/journal
# #159310290 User should be able to sign up

#### What does this PR do?
Implements user signup functionality 

#### Description of Task to be completed?
- have the following endpoint working
`POST/api/v1/auth/users`

#### How should this be manually tested?
- clone the repo `git clone git@github.com:Victor-Ugwueze/journal.git`
- on your terminal run `cd journal`
- run `git checkout ft-user-signup-159310290`
- run `npm install` 
- run `npm run start:dev`

- make a request to `POST localhost:3000/api/v1/auth/users`
 *NB*  see screenshot below on the request data to pass

#### Any background context you want to provide?
- NA

#### What are the relevant pivotal tracker stories?
[#159310290](https://www.pivotaltracker.com/story/show/165170446, "user sign up")

#### Screenshots (if appropriate)
<img width="1680" alt="Screenshot 2019-04-15 at 9 35 32 PM" src="https://user-images.githubusercontent.com/24890605/56163557-81f52200-5fc6-11e9-948d-f7b1ebc925fe.png">

#### Questions:
